src/WebAPI/Controllers/FilmsController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Models;
+
+namespace WebAPI.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    public class FilmsController : Controller
+    {
+        // GET /films/now-playing
+        [HttpGet]
+        [Route("now-playing")]
+        public IEnumerable<FilmModelResponse> GetNowPlayingFilms()
+        {
+            List<FilmModelResponse> nowPlayingFilms = new List<FilmModelResponse>();
+            return nowPlayingFilms;
+        }
+
+        // GET /films
+        [HttpGet]
+        [Route("")]
+        public IEnumerable<FilmModelResponse> Get()
+        {
+            List<FilmModelResponse> Films = new List<FilmModelResponse>();
+            return Films;
+        }
+
+        // GET /films/{name}
```

Comment:
Have you tested this endpoint together with // GET /films/now-playing? I think one of them won't work.